linera-rpc/src/grpc/conversions.rs

```diff
@@ -903,6 +907,13 @@ pub mod tests {
                 hash: CryptoHash::new(&Foo("".into())),
             },
         };
-        round_trip_check::<_, api::Notification>(notification);
+        let message = api::Notification::try_from(notification.clone()).unwrap();
```

Comment:
I was told we should be using `expect(<reason>)` rather than unwraps :)

Reply:
That one is in a test though :)